src/Views/View.php

```diff
@@ -16,9 +16,17 @@ abstract class View
     protected string $footerPath;
     protected string $mainPath = '';
 
+    protected bool|null $isAuth;

```

Comment:
для типа bool, тем более isAuth не особо понятно зачем еще 3е состояние (null) - он либо аутентифицирован, либо нет
поэтому 
```suggestion
    protected bool $isAuth = false;

```

Reply:
+

Reply:
вообще удалил isAuth